doc/tutorials/using_windows_as_uos.rst

```diff
@@ -335,8 +335,8 @@ together with the script used to install Windows 10.
 
 Boot Windows with GVT-g on ACRN
 ===============================
-#. Launch the Windows Guest using the ``launch_igx-waag.sh``. You should see the WaaG desktop
-   coming up over the HDMI monitor (instead of the VNC).
+#. Launch the Windows Guest using the ``launch_win.sh`` in ``/usr/share/acrn/samples/nuc/``.
```

Comment:
launch_win.sh is just a template. Please add note that changes are required for the img path/ovmf path...

Reply:
Sure, it's separated to several steps to let user launch the windows after modify the script.